src/app/components/result/result.component.ts

```diff
@@ -92,21 +92,12 @@ export class ResultComponent implements OnInit, OnChanges {
 
   public openModal(content) {
     this.modalService.open(content).result.then((result) => {
-      this.closeResult = `Closed with: ${result}`;
+      console.log(result)
     }, (reason) => {
-      this.closeResult = `Dismissed ${this.getDismissReason(reason)}`;
+      console.log(reason);
     });
   }
-
-  private getDismissReason(reason: any): string {
-    if (reason === ModalDismissReasons.ESC) {
-      return 'by pressing ESC';
-    } else if (reason === ModalDismissReasons.BACKDROP_CLICK) {
-      return 'by clicking on a backdrop';
-    } else {
-      return  `with: ${reason}`;
-    }
-  }
+$
```

Comment:
Just wondering if this dollar sign has any meaning or if it lies here by mistake